builder/builder.go

```diff
@@ -145,31 +150,43 @@ func (b *Builder) CreateDefaultConfig(localrpms bool) error {
 }
 
 // createRpmDirs creates the RPM directories
```

Comment:
Need to update comment.

Reply:
Boooo, this got lost in my rebase. Thanks for catching!